### ISSUE TITLE
Activation map dimension sizes

### ISSUE THREAD

**Issue by mfatih7** [pytorch/xla#6429]
Hello

For an activation map of BCHW, should we choose sizes of the dimensions as powers of 2 to increase TPU memory utilization efficiency or decrease training time?
Or are we free to choose any number as the size of the dimensions?

**Comment by JackCaoG** [comment]
Rule of thumb on TPU is try to make the dimensions to be a multiple of 8(and 128) to reduce the padding. If you are trying to maximizing the throughout, increasing the memory utilization usually help(larger batch size etc).
